lib/config.ts

```diff
@@ -19,7 +19,6 @@ const rinkeby: Config = {
   chainId: 4,
   nftBackedLoansSubgraph:
     'https://api.thegraph.com/subgraphs/name/with-backed/backed-protocol-rinkeby',
-  pirschCode: 'YJk8KXD6e5xIgVxoe6f9A1O1lSxQW3ky',
```

Comment:
we don't need these in the configs, we really want a staging/prod split for analytics so we'll go back to the env var for that